infra/test/integration/simple_example/simple_example_test.go

```diff
@@ -58,6 +58,39 @@ func TestSimpleExample(t *testing.T) {
 			return false, nil
 		}
 		utils.Poll(t, isServing, 10, time.Minute*1)
+
+		// Check cloud run jobs status
```

Comment:
issue: capitalize product names: "Cloud Run"

Reply:
fixed in [12c25e4](https://github.com/GoogleCloudPlatform/terraform-large-data-sharing-golang-webapp/pull/19/commits/12c25e45d7043e134d12b0d5b91dd4002c41d660)